db/migrate/20160331232622_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```

Comment:
No, unfortunately, I did not have the correct 'master' when I first merged into my account and I had already 'migrated' Devise...my mistake.